lib/simple_form.rb

```diff
@@ -265,6 +265,49 @@ def self.setup
     @@configured = true
     yield self
   end
+
+  # Includes a component to be used by Simple Form. Methods defined in a
+  # component will be exposed to be used in the wrapper as Simple::Components
+  #
+  # Examples
+  #
+  #    # The application needs to tell where the components will be.
+  #    Dir[Rails.root.join('config/initializers/simple_form_components/**/*.rb')].each { |f| require f }
+  #
+  #    # Create a custom component in the path specified above.
+  #    # config/initializers/simple_form_components/input_group.rb
+  #    module InputGroup
+  #      def preprend
+  #        ...
+  #      end
+  #
+  #      def append
+  #        ...
+  #      end
+  #    end
+  #    
+  #    SimpleForm.setup do |config|
+  #      # Create a wrapper using the custom component.
+  #      config.wrappers :input_group, tag: :div, error_class: :error do |b|
```

Comment:
Out of curiosity, what does `b` stand for? 🤔

Reply:
`b` is the wrapper builder.

You can see more about it here:
https://github.com/plataformatec/simple_form/blob/d32cf5fab7e8d2da7eacd98fa8a00017ca1bf07a/lib/simple_form/wrappers/builder.rb#L4